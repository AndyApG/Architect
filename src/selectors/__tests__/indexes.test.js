/* eslint-env jest */

import testState from '../../__tests__/testState.json';

import {
  getVariableIndex,
} from '../indexes';

describe('indexes selectors', () => {
  describe('getVariableIndex()', () => {
    it('extracts variables into index', () => {
      const subject = getVariableIndex(testState);

      expect(subject).toMatchSnapshot();
    });
  });
});
