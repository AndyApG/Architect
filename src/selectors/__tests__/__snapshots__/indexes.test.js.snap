// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`indexes selectors getVariableIndex() extracts variables into index 1`] = `
Object {
  "stages[0].prompts[0].form.fields[0].variable": "alpha",
  "stages[0].prompts[0].form.fields[1].variable": "1234-1234-4",
  "stages[1].prompts[0].form.fields[0].variable": "bravo",
  "stages[1].prompts[0].form.fields[1].variable": "1234-1234-5",
}
`;
